helm/reana/templates/reana-config.yaml

```diff
@@ -12,9 +15,9 @@ data:
     client_pyvenv: {{ .Values.components.reana_ui.client_pyvenv | quote | default "null" }}
     forum_url: {{ .Values.components.reana_ui.forum_url | quote | default "null" }}
```

Comment:
If we update the default value for the forum and docs URLs in the README, shouldn't we update them here as well?

Reply:
They are already set in the values file, I am just keeping the docs up to date: 
https://github.com/reanahub/reana/blob/79aedb9ef2ba0c8a933fe2cac334bd51cd32dd85/helm/reana/values.yaml#L76-L77